components/ray/ray.go

```diff
@@ -69,6 +69,8 @@ func (r *Ray) ReconcileComponent(cli client.Client, owner metav1.Object, dscispe
 		}
 
 		if dscispec.DevFlags.ManifestsUri == "" || len(r.DevFlags.Manifests) == 0 {
+			// inject namespace for SCC
+			imageParamMap["ray-application-namespace"] = dscispec.ApplicationsNamespace
```

Comment:
nit: `imageParamMap` -> `paramMap` or `params` as it's not restricted to the image parameter anymore.

Reply:
yep. either we rename this variable to something generic, like ParamMap, or keep imageParamMap and create another new variable like namesapceParamMap.

want to get a first pass, see what will be a good way

Reply:
did the change: rename it to `ParamMap`